### PR TITLE
PMM-7925: add podman support

### PR DESCRIPTION
**Pull request by denisok**
https://jira.percona.com/browse/PMM-7925

ToDo:

- [ ] test and update Backup and Restore procedure
- [ ] fix links to other pages in pmm-doc
- [ ] update links to the user services when location is known

### CODE REVIEW

**Comment by cezmunsta** [comment]
@denisok nothing stood out, LGTM

File: docs/setting-up/index.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Choose from:
 | Use | <i class="uil uil-thumbs-up"></i> **Benefits** | <i class="uil uil-thumbs-down"></i> **Drawbacks**
 |---|---|---
 | [Docker] | 1. Quick.<br>2. Simple. | 1. Docker installation required.<br>2. Additional network configuration required.
+| [Podman] | 1. Quick.<br>2. Simple.<br>3. Rootless. | 1. Podman installation required.
```
Comment (rnovikovP) [inline docs/setting-up/index.md:19]: we need to make it clear - this is a "Technical review," possible changes in future

File: docs/setting-up/server/podman.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
+# Podman
```
Comment (rnovikovP) [inline docs/setting-up/server/podman.md:1]: we need to make it clear - this is a "Technical review," possible changes in future

**Comment by denisok** [comment]
merged as part of https://github.com/percona/pmm-doc/pull/827

**Pull request closed without merging**